Modules/Core/include/selxAccepting.hxx

```diff
@@ -127,8 +127,8 @@ template< typename FirstInterface, typename ... RestInterfaces >
 bool
 Accepting< FirstInterface, RestInterfaces ... >::AreAllAccepted()
```

Comment:
Yes, I see your point. Lets discuss that later. To give a remark already, this function implements a helper method for the default implementation of ConnectionsSatisfied(). The default requires that all accepting interfaces to be connected, and should be overridden if not required. By overriding one propably would not call AreAllAccepted(). 
We could consider to rename ConnectionsSatisfied() to OK(), but that seems a bit weird since it is a method of the component (not the accepting interfac). ConnectionsOK()?